src/contract_wrappers/registries/module_registry_wrapper.ts

```diff
@@ -169,6 +170,12 @@ interface TagsByModule {
   module: string;
   tags: string[];
 }
+
+interface FactoryDetails {
```

Comment:
Porperty names are a bit off.

I suggest something like

```
  isVerified: boolean;
  ownerAddress: string;
  securityTokenAddresses: string[];
```

I removed `factory` from the first two because the interface is already called `FactoryDetails`, so we know these things refer to the factory.

The third one was too vague and kind of grammatically incorrect. I would also add a minor comment explaning that one in particular. Something like `List of addresses of security tokens that use this module factory`

Reply:
I have resolved these issues in the repo